osc-server/src/test/java/org/osc/core/broker/service/servicefunctionchain/UpdateServiceFunctionChainServiceTest.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package org.osc.core.broker.service.servicefunctionchain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.osc.core.broker.service.dto.BaseDto;
+import org.osc.core.broker.service.request.AddOrUpdateServiceFunctionChainRequest;
+import org.osc.core.broker.service.response.BaseJobResponse;
+
+@RunWith(MockitoJUnitRunner.class)
+public class UpdateServiceFunctionChainServiceTest extends BaseServiceFunctionChainServiceTest {
```

Comment:
i would add more test around the order of vs'es and making sure updates in order are reflected correctly.

Reply:
Yes we should I need to add more persistent Data . Can I add TODO for this test case for now?.

Reply:
Will figure it out how to add more persistent data with out taking to many lines of code. 
Will Add an action Item on it.

Reply:
Added a test case shuffling of updates , but need some ramp up on getting results back and test the order. I have manually tested this scenario throu APIs. Checked it as TODO for that test case.